Content.Server/MassMedia/Systems/NewsSystem.cs

```diff
@@ -324,4 +346,51 @@ private void OnRequestArticleDraftMessage(Entity<NewsWriterComponent> ent, ref N
     {
         UpdateWriterUi(ent);
     }
+
+    #region Discord Hook
+
+    private void OnRoundEndMessageEvent(RoundEndMessageEvent ev)
+    {
+        var query = EntityManager.EntityQueryEnumerator<StationNewsComponent>();
+
+        while (query.MoveNext(out _, out var comp))
+        {
+            foreach (var article in comp.Articles)
+            {
+                SendDiscordWebhook(article);
+            }
+        }
+    }
+
+    private async void SendDiscordWebhook(NewsArticle article)
+    {
+        if (_webhookId is null)
+            return;
+
+        try
+        {
+            var embed = new WebhookEmbed
+            {
+                Title = article.Title,
+                Description = article.Content,
```

Comment:
also you need to respect the 4096 character limit for an embed description

Reply:
oh i see, fine here then (i would leave a comment explaining why this input can be trusted)